build.gradle

```diff
@@ -17,3 +21,9 @@ test {
         events "passed", "skipped", "failed"
     }
 }
+
+javafx{
+    modules = ['javafx.controls', "javafx.fxml"]
+    version = '11.0.2'
```

Comment:
Are you still using javafx?

Reply:
we changed to swing so we dont need this

Reply:
Because it was made by the first commit, and I can't delete the whole build.gradle file. So just leave it this way, it won't do anything.